frontend/src/components/MeetingRoom/RightPanel/RightPanel.tsx

```diff
@@ -16,7 +17,7 @@ export type RightPanelProps = {
  * The panel displays participant list or chat tab.
```

Comment:
Could we add the missing panels? We should have report issue and background effects listed.